cves/CVE-2014-0482.yml

```diff
@@ -142,8 +152,9 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    This vulnerability requires understanding of authentication requirements for RemoteUserMiddleware
```

Comment:
It is not clear why this understanding would prevent an automatic tool from detecting this as long as it was designed by someone with this understanding. Maybe elaborate?